cuebot/src/main/java/com/imageworks/spcue/dao/criteria/FrameSearchFactory.java

```diff
@@ -17,27 +17,18 @@
 
 package com.imageworks.spcue.dao.criteria;
 
-import java.util.List;
-
 import com.imageworks.spcue.FrameInterface;
 import com.imageworks.spcue.JobInterface;
 import com.imageworks.spcue.LayerInterface;
-import com.imageworks.spcue.config.DatabaseEngine;
 import com.imageworks.spcue.dao.criteria.postgres.FrameSearch;
 import com.imageworks.spcue.grpc.job.FrameSearchCriteria;
 
+import java.util.List;
```

Comment:
Current adopted style for java imports has used the following groupings:
* java imports
* third party imports
* package imports

@bcipriano I think we should officially adopt a style guide for the project to reference. Thoughts?

Reply:
Agreed on styleguide.

We have [cuebot/code_style_ij.xml](https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/cuebot/code_style_ij.xml) which can be directly imported into IntelliJ to set up such preferences.